src/main/kotlin/io/github/seggan/sf4k/extensions/SlimefunItemUtils.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.seggan.sf4k.extensions
+
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem
+import org.bukkit.inventory.ItemStack
+
+fun ItemStack.isSlimefun() = SlimefunItem.getByItem(this) != null
+fun ItemStack.asSlimefun() = SlimefunItem.getByItem(this)
```

Comment:
This is personal preference, but I prefer explicit return types on functions that use `=` unless it's super obvious.